    final util.h.xy.ct.d \u02cb(util.h.xy.ct.my) throws java.io.IOException
    {
        java.lang.Throwable $r4;
        util.h.xy.ct.d $r1;
        util.h.xy.ct.md r0;
        util.h.xy.ct.my $r2;
        int $i0, $i1;
        java.lang.ArithmeticException $r3;
        boolean $z0;

        r0 := @this: util.h.xy.ct.md;

        $r2 := @parameter0: util.h.xy.ct.my;

        r0 = new util.h.xy.ct.md;

        specialinvoke r0.<util.h.xy.ct.md: void <init>(java.io.InputStream)>($r2);

        $r1 = virtualinvoke r0.<util.h.xy.ct.md: util.h.xy.ct.d \u02cb()>();

        $i0 = <util.h.xy.ct.md: int \u02c8>;

        $i1 = $i0 & 69;

        $i0 = $i0 | 69;

        $i1 = $i1 + $i0;

        $i0 = $i1 % 128;

        <util.h.xy.ct.md: int \u02bb> = $i0;

        $i1 = $i1 % 2;

        if $i1 == 0 goto label1;

        $z0 = 1;

        goto label2;

     label1:
        $z0 = 0;

     label2:
        if $z0 != 0 goto label3;

        return $r1;

     label3:
        $r3 = new java.lang.ArithmeticException;

        specialinvoke $r3.<java.lang.ArithmeticException: void <init>()>();

        throw $r3;

     label4:
        $r4 := @caughtexception;

        throw $r4;

        catch java.lang.Throwable from label3 to label4 with label4;
    }
